docs/solutions/rhel74_image/settfenv.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/local/bin/python3.6
+
+import os
+import configparser
+import argparse
+
+env_list = {
+    "tenancy": "tenancy_ocid",
+    "fingerprint": "fingerprint",
+    "key_file": "private_key_path",
+    "user": "user_ocid",
+    "pass_phrase": "private_key_password",
+    "public_key": "ssh_public_key",
+    "region": "region",
+    "profile": "profile"
+}
+
+argparser = argparse.ArgumentParser()
+argparser.add_argument('--config_path', required=False,
+                       default='~/.oci/config', dest="config_path")
+argparser.add_argument('--profile', required=False,
+                       default='DEFAULT', dest="profile")
+argparser.add_argument('--dest_file', required=False,
+                       default='./tfenv.sh', dest='dest_file')
+argparser.add_argument('--public_key', required=False,
+                       default="~/.ssh/id_rsa.pub", dest='public_key')
+args = vars(argparser.parse_args())
+
+output_env = {}
+output_file = os.open(args["dest_file"], os.O_RDWR |
+                      os.O_TRUNC | os.O_CREAT, 0x0755)
+
+if len(args["config_path"]) <= 0:
+    oci_config_file = "~/.oci/config"
+else:
+    oci_config_file = args["config_path"]
+
+oci_config_file = os.path.expanduser(oci_config_file)
+
+parser = configparser.ConfigParser()
```

Comment:
It would be much simpler to use the OCI Python SDK for this, which already has a class for parsing and the config file. See "config = oci.config.from_file" in the example at https://oracle-cloud-infrastructure-python-sdk.readthedocs.io/en/latest/.